cmd/complainer/main.go

```diff
@@ -15,13 +16,54 @@ import (
 	"github.com/cloudflare/complainer/uploader"
 )
 
+type regexMatcher struct {
+	whitelist []*regexp.Regexp
+	blacklist []*regexp.Regexp
+}
+
+func (r *regexMatcher) Match(name string) bool {
+	for _, regex := range r.blacklist {
+		if regex.MatchString(name) {
+			return false
+		}
+	}
+	for _, regex := range r.whitelist {
+		if regex.MatchString(name) {
+			return true
+		}
+	}
+	return len(r.whitelist) == 0
+}
+
+type regexArrayFlags []*regexp.Regexp
+
+func (a *regexArrayFlags) String() string {
+	var l []string
+	for _, r := range *a {
+		l = append(l, r.String())
+	}
+	return strings.Join(l, ", ")
+}
+
+func (a *regexArrayFlags) Set(value string) error {
+	r, err := regexp.Compile(value)
+	if r != nil {
+		*a = append(*a, r)
+	}
+	return err
+}
+
 func main() {
 	name := flags.String("name", "COMPLAINER_NAME", monitor.DefaultName, "complainer name to use (default is implicit)")
 	d := flags.Bool("default", "COMPLAINER_DEFAULT", true, "whether to use implicit default reporters")
 	u := flags.String("uploader", "COMPLAINER_UPLOADER", "", "uploader to use (example: s3aws,s3goamz,noop)")
 	r := flags.String("reporters", "COMPLAINER_REPORTERS", "", "reporters to use (example: sentry,hipchat,slack,file)")
 	masters := flags.String("masters", "COMPLAINER_MASTERS", "", "list of master urls: http://host:port,http://host:port")
 	listen := flags.String("listen", "COMPLAINER_LISTEN", "", "http listen address")
+	var whitelist regexArrayFlags
```

Comment:
Offhand, the response is unnecessarily harsh, and you're missing why this was done- the goal is to allow multiple whitelist's to be specified, and to avoid the invoker having to guess whether or not the value is going to be split on comma's.

Simply put, what was used here should be better for the end user.  If you disagree, please clarify why, and we'll proceed from there.